draft-ietf-taps-interface.md

```diff
@@ -1904,47 +1904,70 @@ options.
 ### Capacity Profile {#prop-cap-profile}
 
 Classification:
-: Intent \[TODO: Discuss]
+: Selection Property
 
 Type:
 : Enumeration
 
 Applicability:
 : Preconnection, Connection, Message
 
-This property specifies the application's expectation of the dominating traffic pattern for this Connection.
-This implies that the transport system should optimize for the capacity profile specified. This can influence path and protocol selection.
-The following values are valid for the Capacity Profile:
+This property specifies the traffic interaction pattern expected by the
+application, the desired network treatment for traffic sent by the application,
```

Comment:
I think Gorry took issue with "pattern expected by the application" in #142 (i.e., I don't think this re-wording fixes his problem with the old version). Specifically, he wrote that he'd prefer: "application’s expectation of the treatment the network should provide for..."

Reply:
ah, yep, will fix.